### DISCUSSION TITLE
Paperless-ngx : how choose which HDD drive to install the LXC

### DISCUSSION THREAD

**Discussion by altanir84** [tteck/Proxmox#2166] [Help Needed] [answered]
Hi. I've got two HDDs on my proxmox host, how do I choose which drive to use when installing the LXC?

The secondary drive is in a LVM-thin pool.

Thanks in advance.

**Comment by tteck** [comment, marked as answer]
If both drives are set to 'Container' in the Datacenter, a dialog box will prompt you to choose.


![Screenshot 2023-12-07 5 45 26 PM](https://github.com/tteck/Proxmox/assets/21342670/35f20e53-f5df-418a-8a2a-50f452f0a6fd)

![Screenshot 2023-12-07 5 52 19 PM](https://github.com/tteck/Proxmox/assets/21342670/174ee673-a5b2-4c8c-b92f-ab107921d2e5)

**Comment by altanir84** [comment]
Thank you for your swift reply. I originally tried the advanced way, I thought it was the way to go. 
Got it working using the default settings. Thank you again.
